render.yaml

```diff
@@ -30,4 +30,16 @@ services:
     envVars:
       - key: NODE_VERSION
         value: 16
+
+  - type: web
+    name: npm-stats
+    env: node
+    repo: https://github.com/mui/mui-toolpad.git
+    rootDir: examples/npm-stats
+    buildCommand: yarn && yarn build
+    startCommand: yarn start
+    pullRequestPreviewsEnabled: true
+    envVars:
+      - key: NODE_VERSION
+        value: 16
     # plan: standard
```

Comment:
Is the default plan the `free` plan? If we want reasonable cold start times we'll probably have to specify a plan

Reply:
Changed it to standard for this app.